packages/@react-types/textfield/src/index.d.ts

```diff
@@ -30,7 +30,7 @@ import {
 } from '@react-types/shared';
 import {ReactElement} from 'react';
 
-export interface TextFieldProps extends InputBase, Validation<string>, HelpTextProps, FocusableProps, TextInputBase, ValueBase<string>, LabelableProps {}
+export interface TextFieldProps<T = HTMLInputElement> extends InputBase, Validation<string>, HelpTextProps, FocusableProps<T>, TextInputBase, ValueBase<string>, LabelableProps {}
```

Comment:
can also do because text field should only be input elements
```suggestion
export interface TextFieldProps<> extends InputBase, Validation<string>, HelpTextProps, FocusableProps<HTMLInputElement>, TextInputBase, ValueBase<string>, LabelableProps {}
```

Reply:
I think your initial change is more correct.
I'm not sure that the comment holds true, AriaTextFieldProps extends TextFieldProps, and AriaTextFieldProps is used by useTextField, which allows either input or textarea

Reply:
got it,